.travis.yml

```diff
@@ -27,7 +27,7 @@ env:
     - MOLECULE_NO_LOG=false
     - AWS_REGION=eu-west-1
     - MOLECULE_IT_AWS_VPC_SUBNET_ID=subnet-6bdd4223
-    - DTAS_VERSION=v1.1.1
+    - DTAS_VERSION=master
```

Comment:
we need to make a release before merging if tests are passing

Reply:
The next planned release is 2.0 and it's not prioritized. And I'd rather not release for such a small/quick change